src/targets/cpu/include/migraphx/cpu/dnnl.hpp

```diff
@@ -7,14 +7,26 @@
 #include <migraphx/register_op.hpp>
 #include <migraphx/check_shapes.hpp>
 #include <unordered_map>
-#include <dnnl.hpp>
 #include <migraphx/errors.hpp>
 #include <migraphx/assert.hpp>
+#ifdef MIGRAPHX_ENABLE_ZENDNN
+#include <zendnn.hpp>
+#else
+#include <dnnl.hpp>
```

Comment:
`dnnl.hpp` including itself ? I guess not, can you add a comment here where it is coming from ?

Reply:
This file is `<migraphx/cpu/dnnl.hpp>`, and it's referencing the actual OneDNN and ZenDNN headers here.